### DISCUSSION TITLE
Anzeige Grouplayer im TOC bei Maßstabsgrenzen

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#103] [Ideas]
Hallo Jürgen!

Wäre es möglich/sinnvoll einen Grouplayer im TOC grau darzustellen, wenn die in der Gruppe befindlichen Layer anhand von Maßstabsgrenzen nicht sichtbar (ausgegraut) sind?
Der Wunsch der Abteilung wäre es, ohne den Grouplayer aufklappen zu müssen schon zu erkennen, ob ein Layer innerhalb der Gruppe in der Karte angezeigt wird oder aufgrund vom Maßstab ausgeblendet ist. 

Beispiel im Screenshot: Die Bezeichnung der Grouplayer **Ablagerungen/Kontamination** bzw. **Abwasseranlage** soll ausgegraut werden, da die in der Gruppe vorhandenen Layer aufgrund der Maßstabsgrenzen nicht angezeigt werden. Erst wenn ein Layer sichtbar wird, soll auch der Grouplayer in schwarz dargestellt werden. 

![wis_grouplayer](https://github.com/gis-eni/webgis-forum/assets/122986367/5eb62f94-0b78-492a-a341-f6322a84c7ab)

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Gelöst, kommt mit Build 5.23.3901
